xray/commands/offlineupdate/offlineupdate.go

```diff
@@ -26,9 +26,14 @@ const (
 	JxrayDefaultBaseUrl = "https://jxray.jfrog.io/"
 	JxrayApiBundles     = "api/v1/updates/bundles"
 	JxrayApiOnboarding  = "api/v1/updates/onboarding"
+	periodicState       = "periodic"
+	onboardingState     = "onboarding"
 )
 
 func OfflineUpdate(flags *OfflineUpdatesFlags) error {
+	if flags.IsDBSyncV3 {
+		return handleDBSyncV3OfflineUpdate(flags)
+	}
 	updatesUrl, err := buildUpdatesUrl(flags)
```

Comment:
Your suggestion is pretty much the same as the implementation besides that you request the `if` statement to check against version.

The feature currently is going to be controlled by a feature flag, so having an Xray version dependency is a later stage (when it becomes stable) refactor.

When we will know which Xray version this will be officially released, we will issue a new PR adding that comment

Reply:
All I'm asking for here is a simple refactoring of the above jfrog-cli-core code to make it easier to read.

Reply:
can you suggest the edit you are thinking about? i am not sure i understand.

i wanted to leave the `if` statement here because this is the first entrypoint from the `jfrog-cli` project. i didn't want to pollute both places.

Reply:
done